src/templates/upload.html

```diff
@@ -47,4 +47,4 @@ <h2>You have no uploads!</h2>
 {% else %}
 <p id="description">This shouldn't be possible.  You're not authorized but you are logged in?  wacky.</p>
 {% endif %}
-{% endblock %}
\ No newline at end of file
+{% endblock %}
```

Comment:
why are you deleting the endblock tag

Reply:
will says that is a git moment

Reply:
Git moment, there is still an endblock tag in after merge so unless there is a syntax error i think its fine